pipelines/CiHsc.yaml

```
@@ -29,3 +29,14 @@ tasks:
       doApplyExternalPhotoCalib: False
       doApplyExternalSkyWcs: False
       doApplySkyCorr: False
+  createFakes: lsst.ap.pipe.createApFakes.CreateRandomApFakesTask
+  procWithFakes:
+    class: lsst.pipe.tasks.processCcdWithFakes.ProcessCcdWithFakesTask
+    config:
+      insertFakes.doSubSelectSources: True
+      python: config.insertFakes.sourceSelectionColName = "isVisitSource"
```

Try to avoid using the `python` key unless absolutely necessary.
```suggestion
      insertFakes.sourceSelectionColName: isVisitSource
```

---

Thanks, was playing around to find out what worked and left this in. This pull request isn't going to be merged as it was just a way to for me to test the pipeline with data/a repo.